_config.yml

```diff
@@ -2,16 +2,16 @@
 # Values for this lesson.
 #------------------------------------------------------------
 
-# Which carpentry is this ("swc", "dc", "lc", or "cwl")?
-carpentry: "cwl"
+# Which carpentry is this ("swc", "dc", or "lc")?
+carpentry: "swc"
```

Comment:
LGTM

I had something similar in mind too. Eventually we should phase out all of the carpentry-related stuff, but an approach like this is less dangerous than trying to do it all at once.